src/async_impl/client.rs

```diff
@@ -498,7 +498,10 @@ impl ClientBuilder {
                         config.tls_info,
                     )
                 }
-                #[cfg(feature = "__rustls")]
+                #[cfg(any(
+                    feature = "__rustls_crypto_ring",
+                    feature = "__rustls_crypto_aws_lc-rs"
```

Comment:
```suggestion
                    feature = "__rustls_crypto_aws_lc"
```

Feature name is wrong